chroma_core/lib/service_config.py

```diff
@@ -522,6 +522,9 @@ def _setup_pgsql(self, database, check_db_space):
 
             log.info("Creating database '%s'...\n" % database["NAME"])
             self.try_shell(["su", "postgres", "-c", "createdb -O %s %s;" % (database["USER"], database["NAME"])])
+
+            self.try_shell(["su", "postgres", "-c", "psql -c 'CREATE EXTENSION IF NOT EXISTS btree_gist;' -d %s" % database["NAME"]])
```

Comment:
do you not also need the user here as in the modifications above?

Reply:
No, we log in as `postgres` user as it has administrative privileges required to enable extensions.